pkg/agent/globalstate.go

```diff
@@ -432,7 +440,10 @@ type podStatusDump struct {
 	EndState          *podStatusEndState  `json:"endState"`
 	PreviousEndStates []podStatusEndState `json:"previousEndStates"`
 
-	LastSuccessfulMonitorComm *time.Time `json:"lastSuccessfulMonitorComm"`
+	LastSuccessfulMonitorComm     *time.Time `json:"lastSuccessfulMonitorComm"`
+	FailedMonitorRequestCounter   uint       `json:"failedMonitorRequestCounter"`
+	FailedNeonVMRequestCounter    uint       `json:"failedNeonVMRequestCounter"`
+	FailedSchedulerRequestCounter uint       `json:"failedSchedulerRequestCounter"`
```

Comment:
I wonder if these should have more information — the general philosophy behind these "state dumps" is that it exposes as much internal state as possible, so it might make sense to include the raw timestamps here instead of or in addition to the count.

WDYT?

Reply:
I don't think this info  will be very useful: we don't store the VM names near the timestamps.

Reply:
I agree with @Omrigan. We can add timestamps and VMs names both but I don't see it's going to be helpful. I suggest keeping it simple for now.